src/Lykke.Service.ChainalysisProxy/Controllers/ChainalysisController.cs

```diff
@@ -13,10 +15,12 @@ namespace Lykke.Service.ChainalysisProxy.Controllers
     public class ChainalysisController : Controller
     {
         private readonly IChainalysisProxyService _service;
+        private readonly ILog _log;
 
-        public ChainalysisController(IChainalysisProxyService service)
+        public ChainalysisController(IChainalysisProxyService service, ILog log)
         {
             _service = service;
```

Comment:
Check for null
_service = service ?? throw new ArgumentNullException(nameof(service));
_log = log ?? throw new ArgumentNullException(nameof(log));